client/src/components/AdvisorReports/Toolbar.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react'
+import { Button } from 'react-bootstrap'
+
+class Toolbar extends Component {
+    constructor(props) {
```

Comment:
I would add here the propTypes. That way you force certain property types and you also document the component better.